pkg/agent/packetcapture/capture/bpf_test.go

```diff
@@ -115,7 +116,39 @@ func TestCalculateInstructionsSize(t *testing.T) {
 			count:     15,
 			direction: crdv1alpha1.CaptureDirectionSourceToDestination,
 		},
-
+		{
+			name: "proto with port and flags",
+			packet: &crdv1alpha1.Packet{
+				Protocol: &testTCPProtocol,
+				TransportHeader: crdv1alpha1.TransportHeader{
+					TCP: &crdv1alpha1.TCPHeader{
+						Flags: []crdv1alpha1.TCPFlagsMatcher{
+							{Value: 2},
+							{Value: 16},
```

Comment:
I think it would be slightly more readable if you use an hexadecimal representation
You should also add a comment to indicate which flags you are matching in each case